subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -157,6 +159,12 @@ static int get_job_execution(struct mqtt_client *const client,
 		LOG_ERR("Error when getting the payload: %d", err);
 		return err;
 	}
+
+	char job_doc[payload_len + 1];
```

Comment:
Hide this in an if debug enabled clause? Since it's not used further?

Reply:
good idea